src/store/ChartStore.js

```diff
@@ -224,6 +224,20 @@ class ChartStore {
         this.holderStyle = stxx.chart.panel.holder.style;
 
         stxx.append('deleteHighlighted', this.updateComparisons);
+
+        stxx.append('adjustPanelPositions', function () {
+            let index = 0;
+            for (const x in stxx.panels) {
+                // Hide arrow up for first study
+                if (index == 1) {
+                    const panel = stxx.panels[x];
+                    panel.up.style.display = 'none';
+                    break;
```

Comment:
@MahboobehMohammadi I think this `break` is useless

Reply:
@MahboobehMohammadi you are right. but I think it's not so good to make a loop then break it because u just need index 0 and 1.